### DISCUSSION TITLE
Avalanche core wallet integration

### DISCUSSION THREAD

**Discussion by KaelSensei** [rainbow-me/rainbowkit#721] [Ideas]
Would it be possible to natively integrate the Avalanche Core wallet (the one with the owl logo)?
If not, could you provide support to integrate it by hand?

**Comment by gergelylovas** [comment]
Created a pull request to add support for Core: https://github.com/rainbow-me/rainbowkit/pull/1295

**Comment by DanielSinclair** [comment]
This is available as of [1.0.6](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.0.6)
